keylime/cloud_agent.py

```diff
@@ -254,23 +256,23 @@ def do_POST(self):
                 logger.warning("Unable to decrypt previous payload %s with derived key: %s"%(enc_path,e))
                 os.remove(enc_path)
                 enc_payload=None
-        
+
         # also write out encrypted payload to be decrytped next time
         if enc_payload is not None:
             with open(enc_path,'w') as f:
-                f.write(self.server.payload)
+                f.write(self.server.payload.decode())
 
         # deal with payload
         payload_thread = None
         if dec_payload is not None:
             tomeasure += dec_payload
```

Comment:
This might need to be changed to 

````python
tomeasure = tomeasure + dec_payload
````

I was getting an error saying that the `object cannot re-sized` otherwise.